configure

```diff
@@ -198,6 +169,14 @@ while [ $# -ne 0 ]; do
             echo "${usage}" 1>&2
             exit 1
             ;;
+        --show-config)
+            if [ ! -f "$builddir/CMakeCache.txt" ]; then
+                echo "Error: no CMake build found under '$builddir'." 1>&2
+                exit 1
+            fi
+            grep -E "^ENABLE_|^ZEEK_|^INSTALL_|^CMAKE_INSTALL_PRE|^CMAKE_C.*_FLAGS|^CMAKE_C.*_COMPILER|^CMAKE_BUILD" "$builddir/CMakeCache.txt" | grep -v ':INTERNAL'
```

Comment:
I've used/ran into LINKER_FLAGS previously that represents LDFLAGS usage:

```
$ cmake -S .  -LAH | grep CMAKE_EXE.*_FLAGS
CMAKE_EXE_LINKER_FLAGS:STRING=-fuse-ld=lld
...
```

Reply:
I added this as part of the merge.